test/SymbolGraph/Relationships/Synthesized/Inputs/RemoteP.swift

```diff
@@ -0,0 +1,9 @@
+/// Some Protocol
+public protocol P {
+    func someFunc()
+
+    /// This one has docs!
+    func otherFunc()
+
+    func bonusFunc()
+}
```

Comment:
Should we also test public default implementations of the protocol or is that not relevant?

e.g.,
```swift
public extension P {
    /// This one also has docs!
    func someFunc()
}
```

and assert that "This one also has docs!" doesn't get inherited. I'm not sure about this one because I think the extension `someFunc()` gets a separate USR than `S`'s `someFunc()`.